Tools/CMake/JASP.cmake

```diff
@@ -56,6 +56,26 @@ if(NOT R_REPOSITORY)
       CACHE STRING "The CRAN mirror used by 'renv' and 'install.packages'")
 endif()
 
+option(JASP_RENV_INSTALL_VERBOSE "Tell renv to show package installation logs" OFF)
```

Comment:
So... Im not sure why we need this var...

During building of JASP we always want `renv` to do that verbosely right?
At least, when would I not want to see the compilation?

Or if we do not always want to see everything then I would only want to see problematic compilation.
But then that again would not need this variable.

I know we talked about this before but I think we were both thinking of different things.

Reply:
>  At least, when would I not want to see the compilation?

Can you do in a terminal

```bash
mkdir temp && cd temp && Rscript -e "renv::init()" && Rscript -e "install.packages('BayesFactor', type = 'source')" &> log.txt

```
this will take 10 minutes or so and `log.txt` will be gigantic. 
I think that people almost always want this enabled, but not when they're installing from scratch/ the very first time they need to build all packages.

Reply:
Oh in case you're too lazy to actually do this (I know I would be) here is the result.

[log.txt](https://github.com/jasp-stats/jasp-desktop/files/9754992/log.txt)

There are other R packages aside from BayesFactor that also produce logs of this size...

Reply:
Well I guess I wouldnt be too interested in seeing any of it, if it compiles and isn't our code anyway.
Especially if it's that much :p

Maybe we should do that?
Show `jasp*` r packages verbose and none of the rest.
Then we just make sure we catch that output (which I think you did already, is that in these PRs?) properly so we can throw it at the developer

Reply:
So that would be ideal, but unfortunately in the current setup, it's all or nothing...